lib/fxpay/settings.js

```diff
@@ -80,12 +83,18 @@
         exports[def] = defaultSettings[def];
       }
     }
-    for (var k in newSettings) {
-      if (typeof exports[k] === 'undefined') {
-        exports.log.error('configure() received an unknown setting:', k);
+    for (var param in newSettings) {
```

Comment:
Hmm, I guess not

On Tuesday, January 13, 2015, Stuart Colville <notifications@github.com>
wrote:

> In lib/fxpay/settings.js
> <https://github.com/mozilla/fxpay/pull/59#discussion-diff-22859335>:
>
> > @@ -80,12 +83,18 @@
> >          exports[def] = defaultSettings[def];
> >        }
> >      }
> > -    for (var k in newSettings) {
> > -      if (typeof exports[k] === 'undefined') {
> > -        exports.log.error('configure() received an unknown setting:', k);
> > +    for (var param in newSettings) {
>
> Is there any reason not to use Object.keys(newSettings).forEach?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/fxpay/pull/59/files#r22859335>.
>


-- 
Sent from Gmail Mobile